tests/unit/test_kwargs/test_multipart_data.py

```diff
@@ -493,31 +487,15 @@ def handler(
         assert response.status_code == HTTP_201_CREATED
 
 
-MAX_INT_POSTGRES = 10
-
-
-@define
-class AddProductFormAttrs:
-    name: str
-    amount: int = field(validator=[instance_of(int), ge(1), lt(MAX_INT_POSTGRES)])
-
-
-class AddProductFormPydantic(BaseModel):
-    model_config = ConfigDict(arbitrary_types_allowed=True)
-    name: str
-    amount: int = Field(ge=1, lt=MAX_INT_POSTGRES)
-
-
 class AddProductFormMsgspec(msgspec.Struct):
     name: str
-    amount: Annotated[int, msgspec.Meta(lt=MAX_INT_POSTGRES, ge=1)]
+    amount: Annotated[int, msgspec.Meta(lt=10, ge=1)]
 
 
-@pytest.mark.parametrize("form_object", [AddProductFormMsgspec, AddProductFormPydantic, AddProductFormAttrs])
```

Comment:
We don't actually need to test the different libraries here. This test was added for #2278, but the discrepancies displayed there were expected and a result of the different strictness of the libraries, a difference we do want to preserve. The reason this test was failing before the fix was simply due to invalid input.